orbit/db2.py

```diff
@@ -1,5 +1,5 @@
 import config
-from peewee import Model, IntegerField, CharField, BooleanField, SqliteDatabase
+from peewee import Model, IntegerField, CharField, SqliteDatabase
```

Comment:
same type of comment as before, this commit should be melded into the commit that first introduced the tables